src/dotnet/APIView/APIViewWeb/Pages/Shared/_CommentFormPartial.cshtml

```diff
@@ -1,8 +1,19 @@
-﻿<div id="js-comment-form-template" class="d-none">
+﻿@using APIViewWeb.Models
+@model HashSet<GithubUser>
+<div id="js-comment-form-template" class="d-none">
     <div class="comment-form border-top new-thread-comment">
         <form data-post-update="comments" class="new-thread-comment-form comment" method="post" asp-controller="Comments" asp-action="Add">
-            <div class="new-comment-content">
+            <div class="new-comment-content position-relative">
```

Comment:
Looks like position: relative already exist in site.css for [new-comment-content](https://github.com/Azure/azure-sdk-tools/blob/57547950e8c84df91bfbe134836bb5aab558c74f/src/dotnet/APIView/APIViewWeb/Client/css/site.scss#L1045) is there any reason to also add it here?

Reply:
Not sure why I added it (must've assumed it didn't have it?). Removed now